src/pipe-args.js

```diff
@@ -24,6 +24,7 @@ module.exports.load = (format) => {
 
       chunks.push(buffer.slice(0, nbytes));
     } catch (err) {
+      if (err.code === 'EOF') break; // HACK: see nodejs/node#35997
```

Comment:
should really try-catch just line 21? Also could the error be thrown even though `nbytes != 0`?

Reply:
> _should really try-catch just line 21_

Maybe, though I'm not sure about which line would throw `EAGAIN` under normal conditions (?)

> _Also could the error be thrown even though `nbytes != 0`?_

I don't think so: it just happens when you attempt a read and there are 0 bytes left.

Reply:
If we're sure `EOF` only gets raised when no bytes are left, then this is fine